envi/codeflow.py

```diff
@@ -158,7 +158,7 @@ def addCodeFlow(self, va, arch=envi.ARCH_DEFAULT):
                 print('parseOpcode error at 0x%.8x: %s' % (va, e))
                 continue
             except Exception as e:
-                # traceback.print_exc()
+                traceback.print_exc()
                 print('parseOpcode error at 0x%.8x: %s' % (va, e))
```

Comment:
i recommend using `logging.warn()` with `exc_info=True` here to simplify things.